### ISSUE TITLE
Fix inconsistent spelling of papi namespaces

### ISSUE THREAD

**Issue by tjcouch-sil** [paranext/paranext-core#320]

**Comment by rolfheij-sil** [comment]
@tjcouch-sil Do we also want to change filenames?
Like `data-provider.service.ts` -> `data-providers.service.ts`?
And do we also want to rename things like `dataProviderService` and `dataProviderSources` and the countless other related variables?

**Comment by irahopkinson** [comment]
IMO you don't have to do that in English. `dataProviderSources` is already pluralized so can stay the same. 'service' is (I think) an uncountable noun, e.g. a utility service such as a telephone service provides telephony to many people but we don't pluralize it, but there are many utility services so we pluralize it then. So `data-provider.service.ts' and `dataProviderService` can stay the same also.

**Comment by tjcouch-sil** [comment]
Agreed; let's leave names that aren't part of the `papi`. This issue is just about renaming `papi` stuff. We can do whatever we want with our internal code.

**Comment by tjcouch-sil** [comment]
Agreed; let's leave names that aren't part of the `papi`. This issue is just about renaming `papi` stuff. We can do whatever we want with our internal code.
